boxes.lisp

```diff
@@ -355,11 +355,16 @@ N.B., Successive calls may return the same object."
             :type (integer 0)
             :accessor row-box-padding)
    (contents :initarg :contents
-             :accessor row-box-contents))
+             :accessor row-box-contents)
+   (align :initarg :align
+          :accessor row-box-align
+          :documentation "Contents can be aligned 'top,
+          'baseline (default), 'middle, or 'bottom."))
```

Comment:
Don’t quote symbols in documentation.  Just capitalize them. Also, use keywords, not interned symbols.